### PR TITLE
[deckhouse] rename release status

### PR DESCRIPTION
**Pull request by yalosev**
## Description
Rename deckhouse release status
Make 2 kind of releases: `Skipped` and `Superseded`

## Why do we need it, and what problem does it solve?
Create more obvious release history

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: dechouse
type: chore
summary: Rename DeckhouseRelease Outdated status to Superseded
```

### CODE REVIEW

File: modules/002-deckhouse/crds/deckhouse-release.yaml [hunk 1]
```diff
@@ -69,8 +69,10 @@ spec:
                   enum:
                     - Pending
                     - Deployed
-                    - Outdated
+                    - Outdated # deprecated
                     - Suspended
+                    - Superseded
+                    - Skipped
```
Comment (shvgn) [inline modules/002-deckhouse/crds/deckhouse-release.yaml:75]: Should we have 'Cancelled' here?

Comment (yalosev) [reply modules/002-deckhouse/crds/deckhouse-release.yaml:75]: What does it mean? `Cancelled`

Comment (shvgn) [reply modules/002-deckhouse/crds/deckhouse-release.yaml:75]: It's how we are going to cancel update, skip it intentionally. I guess, we don't have such a scenario for now.

File: modules/002-deckhouse/hooks/cleanup_deckhouse_releases.go [hunk 2]
```diff
@@ -31,7 +31,7 @@ import (
 
 /*
   This hook handle invalid situation when more then 1 Deployed release exists at the moment:
```
Comment (shvgn) [inline modules/002-deckhouse/hooks/cleanup_deckhouse_releases.go:33]:
```suggestion
  This hook handles invalid situation when more then 1 Deployed release exists at the moment:
```

File: modules/002-deckhouse/hooks/cleanup_deckhouse_releases.go [hunk 3]
```diff
@@ -31,7 +31,7 @@ import (
...
-    Hook move all releases except the latest one to the Outdated state
+    Hook move all releases except the latest one to the Skipped state
```
Comment (shvgn) [inline modules/002-deckhouse/hooks/cleanup_deckhouse_releases.go:34]:
```suggestion
    Hook moves all releases except the latest one to the Skipped state
```

**Review by shvgn — APPROVED** [review]

File: modules/002-deckhouse/crds/deckhouse-release.yaml (hunk 1 above)
Comment (yalosev) [reply modules/002-deckhouse/crds/deckhouse-release.yaml:75]: `Suspended` status covers this kind of a scenario

**Review by shvgn — APPROVED** [review]

**Pull request merged**